docs/platforms/android/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -395,19 +395,23 @@ When the UI transaction is not finished yet, but the user makes a new interactio
 
 _(New in version 6.10.0)_
 
-Time to initial display provides insight into how long it takes your Activities to launch and draw their first frame, by adding a span for each launch of an Activity. The SDK sets the span operation to `ui.load.initial-display` and the span description to the Activity's name followed by `initial display`, for example, `MainActivity initial display`.
+Time to initial display (TTID) provides insight into how long it takes your Activities to launch and draw their first frame, by adding a span for each launch of an Activity. The SDK sets the span operation to `ui.load.initial-display` and the span description to the Activity's name followed by `initial display`, for example, `MainActivity initial display`.
 
 The span starts when each Activity is launched, which is defined as an application launch for the first Activity, and the `onPause` method of the previous Activity for each subsequent Activity launched.
 
 The span finishes after the Activity draws its first frame.
 
 Time to initial display is dependent on having an active transaction bound to the scope. Ideally, it should be used along with [Activity Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#activity-instrumentation), which starts a transaction and binds it to the scope automatically.
 
+The following chart shows how Time to initial display (TTID) and [Time to full display (TTFD)](#time-to-full-display) correlate to transitions between activities:
+
+![](./img/ttid-ttfd-concept.png)
+
 ### Time to Full Display
 
 _(New in version 6.14.0)_
 
-Time to full display provides insight into how long it would take your Activities to launch and load all of their content, by adding a span for each launch of an Activity. The SDK sets the span operation to `ui.load.full-display` and the span description to the Activity's name followed by `full display`, for example, `MainActivity full display`.
+Time to full display (TTFD) provides insight into how long it would take your Activities to launch and load all of their content, by adding a span for each launch of an Activity. The SDK sets the span operation to `ui.load.full-display` and the span description to the Activity's name followed by `full display`, for example, `MainActivity full display`.
```

Comment:
```suggestion
By adding a span for each launch of an activity, time to full display (TTFD) provides insight into how long it would take for your activities to launch and load all of their content. The SDK sets the span operation to `ui.load.full-display` and the span description to the activity's name followed by `full display` - for example, `MainActivity full display`.
```